benches/compressed-snark.rs

```diff
@@ -1,9 +1,9 @@
 #![allow(non_snake_case)]
 
-use bellpepper_core::{num::AllocatedNum, ConstraintSystem, SynthesisError};
 use core::marker::PhantomData;
 use criterion::{measurement::WallTime, *};
 use ff::PrimeField;
+use nova_snark::frontend::{num::AllocatedNum, ConstraintSystem, SynthesisError};
```

Comment:
Nit: this import and the next line's import can be merged into one. Can we please apply this everywhere?

Reply:
will do, thanks!